watchtower/__init__.py

```diff
@@ -45,6 +45,20 @@ def _boto_filter(record):
     return True
 
 
+def _size(msg):
+    # Calculate the byte size of a message - accounting for unicode, and extra payload size
+    return (
+        len(msg["message"].encode("utf-8")) if isinstance(msg, dict) else 1
+    ) + CloudWatchLogHandler.EXTRA_MSG_PAYLOAD_SIZE
+
+
+def _truncate(msg, max_size):
```

Comment:
Same here - Can you move _truncate to be a method of CloudWatchLogHandler to simplify customization by subclassing?